org.restlet.java/org.restlet.ext.jetty/src/main/java/org/restlet/ext/jetty/HttpServerHelper.java

```diff
@@ -17,7 +18,16 @@
 
 /**
  * Jetty HTTP server connector.
- * 
+ *
+ * <table>
+ * <caption>list of supported parameters</caption>
+ * <tr>
+ * <td>http.transport.mode</td>
+ * <td>string</td>
+ * <td>http1</td>
```

Comment:
@thboileau Could we use "HTTP1_1" and "HTTP2" for consistency with the protocol version and client-side parameter?

Reply:
sure, thanks